packages/web-components/src/components/grid/column.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2024, 2024
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { LitElement } from 'lit';
+import { property } from 'lit/decorators.js';
+import { carbonElement as customElement } from '../../globals/decorators/carbon-element';
+import { prefix } from '../../globals/settings';
+
+export type ColumnSpecSimple = `${number}` | `${number}%`;
+
+export type ColumnSpec =
+  | ColumnSpecSimple
+  | `span:${number} start:${number}`
+  | `span:${number} end:${number}`
+  | `start:${number} end:${number}`;
+
+@customElement(`${prefix}-column`)
+class CDSColumn extends LitElement {
+  /**
+   * Specify column size
+   * Keys sm, md or lg
+   *
+   * Values
+   * - N, P, { span:N start:S}, { start: S, end: E}
+   * N = number
+   * P = percentage
+   * S = Start column
+   * E = End column (does not reach e.g. start 1 end 3 is same as start 1 span 2)
+   */
+  @property({ reflect: true, attribute: 'sm' })
+  sm?: ColumnSpec;
```

Comment:
```suggestion
  @property({ reflect: true })
  sm?: ColumnSpec;
```

if the attribute and property name are the same, there's no need to declare the `attribute` since it's inferred from the property name by default